src/components/App/index.js

```diff
@@ -103,9 +104,20 @@ class ModalApp extends Component {
         this.onInvalidJWT('hideOnfidoLogo feature not enabled for this account.')
       }
 
-    } else if (!options.mobileFlow && token && token !== prevToken) {
+    } else if (!options.mobileFlow && isNewToken) {
       this.props.actions.hideOnfidoLogo(false)
     }
+    if (options.enterpriseFeatures?.cobrand && !isHideOnfidoLogoEnabled && isNewToken) {
```

Comment:
The `prepareInitialStore` function is 51 lines and it's full of if else statement. Could we split it into smaller functions to make it more readable?

Reply:
Good point! Will work on splitting it up to be more manageable & readable.